glide.yaml

```diff
@@ -0,0 +1,11 @@
+package: github.com/m3db/m3ctl
+
+import:
```

Comment:
Maybe copy the packages (and their corresponding versions) that are in `glide.lock` but aren't in `glide.yaml` into `glide.yaml` so they don't change when someone runs `glide update`?